main.js

```diff
@@ -23,5 +25,115 @@ options.forEach((e) => {
     wrapper.classList.toggle('p-fixed');
     wrapper.classList.toggle('d-none');
     wrapper.classList.toggle('d-flex');
+    document.body.classList.toggle('no-scroll');
+  });
+});
+
+const dataArray = [
+  {
+    id: 0,
+    imgUlr: './img/workPhoto.png',
+    name: 'Tonic',
+    technologies: ['CANOPY', 'BACK END DEV', '2015'],
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos  pariatur voluptas, amet autem natus quam vitae culpa ut! Voluptate officiis eligendi non atque? Ab necessitatibus, ipsa aliquam provident, odio incidunt eum debitis ipsum non animi consecteturomnis, labore eius quo dolores rerum nobis vel recusandae saepe fuga sed libero perferendis?',
+    language: ['HTML', 'CSS', 'JS'],
+  },
+  {
+    id: 1,
+    imgUlr: './img/workPhoto.png',
+    name: 'Tonic',
+    technologies: ['CANOPY', 'BACK END DEV', '2015'],
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos  pariatur voluptas, amet autem natus quam vitae culpa ut! Voluptate officiis eligendi non atque? Ab necessitatibus, ipsa aliquam provident, odio incidunt eum debitis ipsum non animi consecteturomnis, labore eius quo dolores rerum nobis vel recusandae saepe fuga sed libero perferendis?',
+    language: ['HTML', 'CSS', 'JS'],
+  },
+  {
+    id: 2,
+    imgUlr: './img/workPhoto.png',
+    name: 'Tonic',
+    technologies: ['CANOPY', 'BACK END DEV', '2015'],
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos  pariatur voluptas, amet autem natus quam vitae culpa ut! Voluptate officiis eligendi non atque? Ab necessitatibus, ipsa aliquam provident, odio incidunt eum debitis ipsum non animi consecteturomnis, labore eius quo dolores rerum nobis vel recusandae saepe fuga sed libero perferendis?',
+    language: ['HTML', 'CSS', 'JS'],
+  },
+  {
+    id: 3,
+    imgUlr: './img/workPhoto.png',
+    name: 'Tonic',
+    technologies: ['CANOPY', 'BACK END DEV', '2015'],
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos  pariatur voluptas, amet autem natus quam vitae culpa ut! Voluptate officiis eligendi non atque? Ab necessitatibus, ipsa aliquam provident, odio incidunt eum debitis ipsum non animi consecteturomnis, labore eius quo dolores rerum nobis vel recusandae saepe fuga sed libero perferendis?',
+    language: ['HTML', 'CSS', 'JS'],
+  },
+];
```

Comment:
Wonderful job setting this **_array of objects_** to display the modals. 👍 

However, I would like to point out a few things to you:

- [ ] For this project milestone, it would be nice to use this array created to populate the data for both the `works section (Project Section)` with `id="works"` and the `popup`. In essence, it would require that you remove all _**static**_ data in your `index.html` for the `works section` and use this same array to _**dynamically**_ populate the `works section` and your `popup`. You could use your code written for the creation of the popup as clue for fulfilling this requirement.

- [ ] Each `object` in your array should have a `key` called `links` and it should have two values:
  - A link to the **_live version_** of your project, 
  - A link to the **_project source code_** . 
 This code snippet should give you an idea 
```
 links: [' https://redbull.github.io/Template-Portfolio/', 'https://github.com/Redbull/Template-Portfolio']
```
The links will be used for this part of the popup 
![image](https://user-images.githubusercontent.com/77106962/152542055-ca0561bf-7b55-4d3f-b922-c920ce30c58f.png)